src/Sulu/Bundle/ContentBundle/Resources/views/Template/content.html.twig

```diff
@@ -11,6 +11,7 @@
             {% set type = get_type(property.contentTypeName) %}
 
             {% if property.contentTypeName == 'block' %}
+                {% include 'SuluContentBundle:Template:macros/single.html.twig' with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
                 {% include type.template with {'property': property, 'params': params, 'type': type, 'id': property.name, 'webspaceKey': webspaceKey, 'languageCode': languageCode, 'userLocale': userLocale} only %}
```

Comment:
this line will be called also in `SuluContentBundle:Template:macros/single.html.twig` everything will be rendered twice i think? perhaps its enough to remove it but we should ask @berndhep if he want the title over the blocks?